src/library/autodj/autodjprocessor.cpp

```diff
@@ -1343,6 +1362,7 @@ void AutoDJProcessor::calculateTransition(DeckAttributes* pFromDeck,
     }
 
     switch (m_transitionMode) {
+        // case TransitionMode::RadioFullIntroOutro:
```

Comment:
What is that?

Reply:
As I said elsewhere that is a comment form an earlier experiment, that I neglected to delete.